static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -77,9 +86,14 @@ type PbjsEventHandler = PbjsEventData => void;
 const bidderTimeout: number = 1500;
 
 const consentManagement: ConsentManagement = {
-    cmpApi: 'iab',
-    timeout: 200,
-    allowAuctionWithoutConsent: true,
+    gdpr: {
+        cmpApi: 'iab',
+        timeout: 200,
+        allowAuctionWithoutConsent: true,
+    },
+    usp: {
+        timeout: 8000,
```

Comment:
Why is this timeout so long? Eight seconds seems like an eternity 😆

Reply:
Hah yes! Sourcepoint has advised that, but think we will reduce it something way smaller

Reply:
Sourcepoint's system can take quite a while to load, especially in slower connections. So that's probably why.

Reply:
We have asked them again and they are happy with going down to 1500ms!

Reply:
If the timeout expires before the uspapi is ready, does prebid assume the default state (CCPA opt out is false)?

Reply:
From there docs:
```
If the timeout period expires or an error from the CMP is thrown, the auction proceeds without the user’s consent information.
```

Reply:
@ioanna0, can you put the link to their docs, too, for future travellers?

Reply:
That's not very helpful unfortunately. "`without the user's consent information`" might mean a few different things. It might mean it defaults to the user not giving consent, defaults to whatever is the default state of the applicable framework or leaves it up to the vendor's choice. The last two present problems from a data privacy angle, as a user's data might be used when they haven't given consent.
Happy to chat 'offline' if you'd like.

Reply:
@mxdvl I've included it in the PR's description but here's the link here as well: http://prebid.org/dev-docs/modules/consentManagementUsp.html

Reply:
@ripecosta Yes there's some ambiguity  in there, definitely worth clarifying it. I will check if we can clarify that with the business. I think for now this timeout should be safe enough as it's still quite high and it's the same as the prebid timeout.

Reply:
Not sure how safe it is given that Sourcepoint takes considerably longer to load compared to our TCFv1 implementation. If these are timing out it could be an issue.